Examples/ServiceExamples/Scripts/ExamplePersonalityInsightsV3.cs

```diff
@@ -0,0 +1,172 @@
+﻿/**
+* Copyright 2015 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using UnityEngine;
+using IBM.Watson.DeveloperCloud.Services.PersonalityInsights.v3;
+using IBM.Watson.DeveloperCloud.Logging;
+
+public class ExamplePersonalityInsightsV3 : MonoBehaviour
+{
+	PersonalityInsights m_personalityInsights = new PersonalityInsights();
```

Comment:
Yes, I'm actually doing both. There is string testString and string dataPath. In the Start method we call GetProfile using dataPath then we call GetProfile using the testString. I don't have access to Application.dataPath in the constructor so I need to set it in the Start method. I guess I can define first and set it when we start?